.github/CODEOWNERS

```diff
@@ -579,7 +579,7 @@ x-pack/plugins/serverless @elastic/appex-sharedux
 x-pack/plugins/serverless_observability @elastic/appex-sharedux
 packages/serverless/project_switcher @elastic/appex-sharedux
 x-pack/plugins/serverless_search @elastic/appex-sharedux
-x-pack/plugins/serverless_security @elastic/appex-sharedux
+x-pack/plugins/serverless_security @elastic/security-solution
```

Comment:
I think this is OK for now, but later we will need to split code ownership by the components, subfolders etc. to reduce PR noise across Security Solution teams.